### PR TITLE
Fix keyboard handling in the Rich Text Insert Symbol dialog

### PR DESCRIPTION
**Pull request by oneeyeman1**
Arrow keys and TABs are working properly.

Pressing Space currently doesn't do anything.

Please review and apply.

### CODE REVIEW

File: src/richtext/richtextsymboldlg.cpp [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ void wxSymbolListCtrl::OnPaint(wxPaintEvent& WXUNUSED(event))
 
 void wxSymbolListCtrl::DoHandleItemClick(int item, int WXUNUSED(flags))
 {
-    if (m_current != item)
+//    if (m_current != item)
```
Comment (vadz) [inline src/richtext/richtextsymboldlg.cpp]: This can't be intentional, can it?

Also, could you _please_ review your PRs before submitting them?

Comment (oneeyeman1) [reply src/richtext/richtextsymboldlg.cpp]: @vadz ,
Oops, sorry.

Fixed now.

Thx.

**Comment by vadz** [comment]
@JulianSmart would you like to have a look at this by chance? I have no idea about this code, so reviewing it is rather difficult.

**Comment by oneeyeman1** [comment]
@vadz , since Julian is not very active anymore (unfortunately), can you apply this? This way it will be exposed and get more testing...
Or maybe you have a way to ping Julian?

**Comment by vadz** [comment]
I've merged this in 42bf26730f871bb3887dbda64821413ad3b9f1a0 but forgot to reference this PR.

**Pull request closed without merging**